### ISSUE TITLE
Small Readme update

### ISSUE THREAD

**Issue by hallindavid** [sethsandaru/demo-vue-form-builder#1]
Hi there - just a super small note - love the package, but in the readme, it says 

> Note: to turn on insert & update form config, you need to:
> 
> Open file: config/sandaru.php
> Set disable_populate to **true** (Sorry for this inconvenience, I add this constraint to make sure that my Live Demo doesn't have any trash data in there)

This should actually be 
> Set disable_populate to **false** (Sorry for this inconvenience, I add this constraint to make sure that my Live Demo doesn't have any trash data in there)

**Comment by RobertoMaurizzi** [comment]
It would be nice to have some simple setup instructions... for example, it turns out you can simply run `php artisan serve` to launch a development server and test the site on localhost.

**Comment by sethsandaru** [comment]
Ah yes, sorry for my bad. I'll update it.

@RobertoMaurizzi , You're right. But `php artisan serve` might not enough (I think, or SQLite could work well). 

But I think for the better, I should prepare the README with the database's tables and suggest some best practices instead of using Laravel project. What do you think?

**Comment by RobertoMaurizzi** [comment]
True, but following your current steps then launching `php artisan serve` works and it's much simpler than trying to setup a proper web server+php application server :-)
Sorry for the very late reply!
